src/server/middleware/systraceMiddleware.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ * 
+ * @flow
+ */
+import type { Middleware, $Request } from 'express';
+
+const fs = require('fs');
+const dedent = require('dedent');
+const logger = require('../../logger');
+
+const systraceMiddleware: Middleware = (req: $Request, res, next) => {
+  if (req.path !== '/systrace') {
```

Comment:
It's not configurable. We're still hardcoding it where we define the server. Except we use express's feature instead of doing this manual check in every middleware.